packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet-collapse-wrapper/atomic-smart-snippet-collapse-wrapper.tsx

```diff
@@ -25,6 +25,21 @@ export class AtomicSmartSnippetCollapseWrapper {
 
   private shouldRenderButton = !!this.maximumHeight;
 
+  private validateProps() {
```

Comment:
Sorry, I'm not sure to understand why this validation is now on the atomic-smart-snippet-collapse-wrapper. There is not `snippetMaximumHeight` nor `snippetCollapsedHeight` property on that clas

Reply:
The `snippetMaximumHeight` is added by the user in the atomic-smart-snippet component. Then inside of it it is sent to the atomic-smart-snippet-collapse-wrapper. [link](https://github.com/coveo/ui-kit/blob/KIT-2780/packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet/smart-snippet-common.tsx#L157-L159)

Inside of the collapse wrapper component the prop was simply called maximumHeight instead of snippetMaximumHeight. I just changed the prop name for less confusion and also I think this component is supposed to be @internal . I asked the author about it.

I changed the location of the validation since the error was not functioning correctly (probably because of the atomicSmartSnippetCommon pattern. I followed the same pattern used to validate maximumHeight and collapsedHeight. [link](https://github.com/coveo/ui-kit/blob/KIT-2780/packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet/smart-snippet-common.tsx#L95)